docs/dev-tools/chopsticks/overview.md

```diff
@@ -0,0 +1,412 @@
+    Account:
+      -
+        -
+          - 5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
+        - providers: 1
+          data:
+            free: '10000000000000000000'
+  ParasDisputes:
+    $removePrefix: ['disputes'] # those can makes block building super slow
+```
+
+To run Chopsticks using a configuration file, utilize the `--config` flag. You can use a raw GitHub URL, a path to a local file, or simply the chain's name. For example, the following commands all use Polkadot's configuration in the same way:
+
+=== "GitHub URL"
+
+    ```bash
+    npx @acala-network/chopsticks \
+    --config=https://raw.githubusercontent.com/AcalaNetwork/chopsticks/master/configs/polkadot.yml
+    ```
+
+=== "Local File Path"
+
+    ```bash
+    npx @acala-network/chopsticks --config=configs/polkadot.yml
+    ```
+
+=== "Chain Name"
+
+    ```bash
+    npx @acala-network/chopsticks --config=polkadot
+    ```
+
+!!! note
+    If using a file path, make sure you've downloaded the [Polkadot configuration file](https://github.com/AcalaNetwork/chopsticks/blob/master/configs/polkadot.yml){target=_blank}, or have created your own.
+
+### Using Command Line Interface (CLI)
+
+Alternatively, all settings (except for genesis and timestamp) can be configured via command-line flags, providing a comprehensive method to set up the environment. For example, the following command forks Polkadot at block 100.
+
+```bash
+npx @acala-network/chopsticks \
+--endpoint wss://polkadot-rpc.dwellir.com \
+--block 100
+```
+
+## Interacting with a Fork
+
+When running a fork, it's accessible by default at:
+
+```bash
+ws://localhost:8000
+```
+
+You can interact with the forked chain using various [libraries](https://wiki.polkadot.network/docs/build-tools-index#libraries){target=_blank} such as [Polkadot.js](https://polkadot.js.org/docs/){target=_blank} and its user interface, [Polkadot.js Apps](https://polkadot.js.org/apps/#/explorer){target=_blank}.
+
+### Using Polkadot.js Apps
+
+To interact with Chopsticks via the hosted user interface, visit [Polkadot.js Apps](https://polkadot.js.org/apps/#/explorer){target=_blank} and follow these steps:
+
+1. Click the network icon in the top left corner
+2. Scroll to the bottom and select **Development**
+3. Choose **Custom**
+4. Enter `ws://localhost:8000` in the input field
+4. Click the **Switch** button
+
+![](/polkadot-ecosystem-docs-draft/images/dev-tools/chopsticks/chopsticks-1.webp)
+![](/polkadot-ecosystem-docs-draft/images/dev-tools/chopsticks/chopsticks-2.webp)
+
+You should now be connected to your local fork and can interact with it as you would with a real chain.
+
+### Using Polkadot.js Library
+
+For programmatic interaction, you can use the Polkadot.js library. Here's a basic example:
+
+```js
+import { ApiPromise, WsProvider } from '@polkadot/api';
+
+async function connectToFork() {
+  const wsProvider = new WsProvider('ws://localhost:8000');
+  const api = await ApiPromise.create({ provider: wsProvider });
+  
```

Comment:
I believe it is a good practice to do `await api.isReady` -> https://polkadot.js.org/docs/api/start/create#api-instance

Reply:
Nice one! I didn't know about this